.github/workflows/live-deploy.yml

```diff
@@ -1,9 +1,11 @@
 # An Action to build Web Arcs and deploy to https://live.arcs.dev
-name: live-deploy
+name: arcs-live
 
 on:
   push:
     branches: [ master ]
+  pull_request:
```

Comment:
I don't think we need this. This will try to deploy for every PR?

Reply:
I agree. I'm including this here for testing / iterating purposes only. I will remove this before final merge.

Reply:
Removed.